Microsoft.Toolkit.Uwp.UI/Helpers/Emoji/Emoji.cs

```diff
@@ -0,0 +1,5810 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Helpers.Emoji
+{
+    /// <summary>
+    ///     Enum listing the names of all the emojis
+    /// </summary>
+    public enum Emoji
+    {
+        /// <summary>
+        /// The Helmet With White Cross emoji
+        /// </summary>
+        HelmetWithWhiteCross,
```

Comment:
What happens when a new emoji is added? How will we know what needs to be added?